examples/packer-aws/ubuntu-2204.pkr.hcl

```diff
@@ -0,0 +1,82 @@
+packer {
+  required_plugins {
+    amazon = {
+      version = ">= 1.1.0"
+      source  = "github.com/hashicorp/amazon"
+    }
+    mondoo = {
+      version = ">= 0.2.1"
+      source  = "github.com/mondoohq/mondoo"
+    }
+  }
+}
+
+variable "aws_profile" {
```

Comment:
Could you also add the `mondoo_config_path` var? Nice to give the user the full set of options (also down in the provisioner config)

Reply:
I think we should remove the `aws_profile` variable all together. It will make it easier to configure in CI.

I will add the `mondoo_config_path`

Reply:
@preslavgerchev I never got to test out `mondoo_config_path`. What is the order of precedence between that and the environment variable `MONDOO_CONFIG_PATH`?

Reply:
That's a great question, I just tested this - a config file has precedence over the env var